dts/msm8916-asus-z00l.dts

```diff
@@ -0,0 +1,20 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+/dts-v1/;
+
+/include/ "msm8916.dtsi"
+
+/ {
+        qcom,msm-id =   <206 0>,
+			<248 0>,
+			<249 0>,
+			<250 0>;
+	qcom,board-id = <21 0>;
+	compatible = "qcom,msm8916-mtp", "qcom,msm8916";
+	
+	asus-z00l {
```

Comment:
Your original changes in the first commit were fine, this special syntax is only needed when multiple devices use one board-id.

Reply:
See https://github.com/msm8916-mainline/lk2nd/blob/3f6837d645d36e27a4a0a4298a91e52ce6c0febe/dts/msm8939-asus-z00t.dts#L10 , Asus Z00T is very similar to Z00L but uses a different SOC but the board id seems to be similar.

Reply:
https://github.com/msm8916-mainline/lk2nd/commit/1f5a335422866c8f3e952cfce8ebc8083a85a531